### DISCUSSION TITLE
revit to bim by iTwin Snapshot

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#505] [Q&A]
when i use revit to bim by iTwin Snapshot,
1.the element some property value not is chinese languge for example revit element family value 
2.the element some property name in revit is chinese ,but in bim file is english.

so, who can help me ?

**Comment by pmconne** [comment]
Hi,

Please be aware that most folks who might be able to help you are on holiday until next week.
It would help for you to clarify where exactly you are seeing English instead of Chinese. In the property grid UI? Inside the database file?
Are *all* element property names and/or values in English, or are some in Chinese?
If possible, supply the .bim and/or .rvt files.

**Comment by paprophet** [comment]
thank you！
please give me your email address.i can send the file .
in fact , the revit file is drawing in revit software by  chinese language version ,but change to bim by iTwin Snapshot.
in bim file ,the family name is store in english,and the element system property name is english too, but some indefine element property name is store by uft-8 to chinese ,so,i want is iTwin Snapshot have some config to set ?

**Comment by pmconne** [comment]
I forwarded your concerns to the folks who work on the Revit-to-iModel connector. They are aware of this issue. Next week (after New Year holidays) they will follow up with their plans to address the problem.

**Comment by jffmarker** [comment]
Sorry for the delay. iModels don't currently support changing the display language of schemas at runtime; class names are baked into the iModel by the App creating the schema. One stopgap we've considered is for the Revit Connector to detect the OS language, and on a first-in-wins basis, store the intended language for future reference, and then use that for to localize the Revit schema when first writing it to the iModel. This work is not currently scheduled, however.
